android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -39,9 +39,10 @@ open class Configuration @JvmOverloads constructor(
     var locationListening: Boolean = true,
     var flushEventsOnClose: Boolean = true,
     var minTimeBetweenSessionsMillis: Long = MIN_TIME_BETWEEN_SESSIONS_MILLIS,
-    @Deprecated("Please use 'defaultTracking.sessions' instead.", ReplaceWith("defaultTracking.sessions"))
-    var trackingSessionEvents: Boolean = true,
-    var defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
+    trackingSessionEvents: Boolean = true,
+    @Suppress("DEPRECATION")
+    defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
```

Comment:
why are we removing `@Deprecated` and adding `@Suppress` here? 
```suggestion
    @Deprecated("Please use 'autocapture.sessions' instead.", ReplaceWith("autocapture.sessions"))
    trackingSessionEvents: Boolean = true,
    @Deprecated("Please use 'autocapture' instead.", ReplaceWith("autocapture"))
    defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
```